src/olympia/bandwagon/admin.py

```diff
@@ -87,5 +96,14 @@ def has_add_permission(self, request):
             request.user, amo.permissions.ADMIN_CURATION
         ) or super().has_add_permission(request)
 
+    def delete_model(self, request, obj):
+        ActivityLog.objects.create(amo.LOG.COLLECTION_DELETED, obj)
+        obj.delete(clear_slug=False)
+
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        if change and 'deleted' in form.changed_data and not obj.deleted:
+            ActivityLog.objects.create(amo.LOG.COLLECTION_UNDELETED, obj)
```

Comment:
Same question.

Reply:
Undeletion is not a feature exposed outside of the admin